tests/vk-regression/vk-regression.json

```diff
@@ -203,16 +203,16 @@
     }
   },
   "ecdsa": {
-    "digest": "6e4078c6df944db119dc1656eb68e6c272c3f2e9cab6746913759537cbfcfa9",
+    "digest": "36c90ec17088e536562b06f333be6fc1bed252b47a3cfb4e4c24aa22e7a1247b",
     "methods": {
       "verifyEcdsa": {
-        "rows": 38846,
-        "digest": "892b0a1fad0f13d92ba6099cd54e6780"
+        "rows": 38888,
+        "digest": "70f148db469f028f1d8cb99e8e8e278a"
```

Comment:
the example circuit changed: it previously did no validity checks on the input public key, now it does (automatically, when the input is witnessed)